### PR TITLE
feat(2210): Show parameter description

### PR DESCRIPTION
**Pull request by kumada626**
## Context

Show description of parameter on the UI.
Description is shown by moving cursor on icon.

スクリーンショット 2020-10-09 18 40 43

スクリーンショット 2020-10-09 18 40 04

## Objective

Add an icon to head of the parameter name. If parameter description is defined, parameter description is shown on UI.

## References

screwdriver-cd/screwdriver#2210

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/pipeline-parameterized-build/template.hbs [hunk 1]
```diff
@@ -3,31 +3,41 @@
...
-    {{#if (is-array parameter.value)}}
-      {{#form.group}}
-        <label class="control-label col-md-4">{{parameter.name}}</label>
-        <div class="col-md-8">
+    {{#form.group}}
+      <label class="control-label col-md-4">
+        {{#if parameter.description}}
+          {{fa-icon "question-circle" title=parameter.description}}
+        {{else}}
+          {{fa-icon "question-circle" title="No description."}}
```
Comment (wahapo) [inline app/components/pipeline-parameterized-build/template.hbs:11]: If no description, I think it's not necessary to show the icon.

Comment (kumada626) [reply app/components/pipeline-parameterized-build/template.hbs:11]: I added icon always for preventing break form position. However, current implementation not break form position if without icon.
I remove icon if description is not defined.

**Review by wahapo — APPROVED** [review]

File: app/components/pipeline-parameterized-build/component.js [hunk 2]
```diff
@@ -124,11 +124,7 @@ export default Component.extend({
       }
     },
 
-    onUpdateDropdownValue(model, propertyName, value) {
-      this.updateValue({ model, propertyName, value });
-    },
-
-    onUpdateValue(value, model, propertyName) {
```
Comment (adong) [inline app/components/pipeline-parameterized-build/component.js:131]:
```
onUpdateDropdownValue(model, propertyName, value)

onUpdateValue(value, model, propertyName)
```

Just want to point out these two functions have a different order of parameters

File: app/components/pipeline-parameterized-build/template.hbs [hunk 3]
```diff
@@ -3,31 +3,39 @@
...
-        onChange=(action "onUpdateValue")}}
-    {{/if}}
+        {{else}}
+          {{input
+            value=parameter.value
+            class="form-control"
+            label=parameter.value
+            placeholder=parameter.value
+            property=parameter.name
+            change=(action (action "onUpdateValue" parameterizedModel parameter.name parameter.value))
```
Comment (adong) [inline app/components/pipeline-parameterized-build/template.hbs:34]: Looks like you have noted and updated the action function here. Cool!

**Review by adong — APPROVED** [review]

**Pull request merged**